### DISCUSSION TITLE
I got a question for the /start command

### DISCUSSION THREAD

**Discussion by docteurfr** [sleeyax/aternos-discord-bot#52] [Q&A] [answered]
How can you do when you want player can
![5735-tanjiro-confused](https://user-images.githubusercontent.com/108757494/177387310-c0d3d1db-27fd-4575-92cc-4e8aab9f99a5.png)
 do the command /start ?

**Comment by sleeyax** [comment, marked as answer]
Setup slash command permissions accordingly.

https://discord.com/blog/slash-commands-permissions-discord-apps-bots

**Reply by docteurfr** [reply]
and how can setup slash command ?

**Reply by sleeyax** [reply]
See the link 🙄

**Reply by docteurfr** [reply]
dont work

**Reply by docteurfr** [reply]
can you add me in discord for help me ?

**Reply by docteurfr** [reply]
nvm it work

**Reply by sleeyax** [reply]
> nvm it work

See, all it takes is a little bit of effort.
